staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/scheme/scheme.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package scheme
```

Comment:
Created a new package for using `Scheme` because importing `k8s.io/apiextensions-apiserver/pkg/apiserver` for using `Scheme` creates an import loop:

```
can't load package: import cycle not allowed
package k8s.io/kubernetes/vendor/k8s.io/apiextensions-apiserver/pkg/apiserver
	imports k8s.io/apiextensions-apiserver/pkg/registry/customresourcedefinition
	imports k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/validation
	imports k8s.io/apiextensions-apiserver/pkg/apiserver
```